packages/design-tokens/src/tokens/functional/components/river/river.json

```diff
@@ -12,6 +12,17 @@
           }
         }
       },
+      "innerY": {
+        "value": "var(--base-size-24)",
+        "responsive": {
+          "768px": {
+            "value": "var(--base-size-24)"
+          },
+          "1280px": {
+            "value": "var(--base-size-32)"
```

Comment:
@danielguillan @jesussandreas For breakout vertical spacing, i only have a reference point for desktop. Left mobile and tablets at 24px as before. Let me know if you feel it should also change.